tests/ekore/anomalous_dimensions/unpolarized/time_like/test_as3.py

```diff
@@ -1,3 +1,5 @@
+# it is possible to obtain result values for testing from vbertone/MELA repo
```

Comment:
```suggestion
"""Test values have been obtained running MELA.

MELA is an alternative Mellin space evolution code, hosted at https://github.com/vbertone/MELA.

"""
```

The second part you can put in a single place, and omit from individual test modules. Even in the docs, if you wish :)